sporkfish/searcher/negamax_lazy_smp.py

```diff
@@ -46,12 +46,10 @@ def _start_search_from_root(
         :return: Tuple containing the best move and its value.
         :rtype: Tuple[float, chess.Move]
         """
+        task = lambda: super()._start_search_from_root(board, depth, alpha, beta)
```

Comment:
If I understood correctly this would recursively call itself, infinitely spawning new processes - a disaster! We want to call the parent class function with the same name, just over multiple processes. 

I don't get the hate over lambdas tbh